app/controllers/judges_controller.rb

```diff
@@ -157,47 +156,62 @@ def activity
     @year = params[:year] || Contest.order(:start).last.start.year
 
     # Create a hash of Category.id => Category.name
+    # TODO should this be category and aircat instead of name?
+    #   If a flight was only recorded in a synthetic
+    #   category, that will be the category found
     ch = Category.pluck(:id, :name).to_h
 
     # Get all Contest objects for the year in question
-    Contest.where(['year(start) = ?', @year]).includes(:flights).find_each do |contest|
+    Contest.where(['year(start) = ?', @year]).includes(
+      :flights
+    ).find_each do |contest|
 
       # Convenience var
       nats = (contest.region == "National")
 
       # For each flight (e.g., Known/Free/Unknown)
       contest.flights.find_each do |flight|
+        # report only one category so the flight is not double-counted
+        # order by sequence so the one chosen is a "regular" category
+        # however if the flight was only recorded in a synthetic
+        # category, that will be the category found
+        category = ch[flight.categories.order(:sequence).first.id]
```

Comment:
@djmolny Ah. Here it is. It's sending the name of the first category for the flight. The only effect will be that the "U.S. Advanced Team" category named flight (one, the second unknown) will still not be credited. The consideration is to use the category 'advanced' and aircat 'P' in place of the name, at which time that flight could be credited.